src/Microsoft.Dnx.Runtime.Abstractions/Constants.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Dnx.Runtime
+{
+    public static class RuntimeTypes
+    {
+        public static readonly string CoreCLR = nameof(CoreCLR);
+        public static readonly string CLR = nameof(CLR);
+        public static readonly string Mono = nameof(Mono);
+    }
+
+    public static class RuntimeArchitectures
+    {
+        public static readonly string X86 = nameof(X86);
+        public static readonly string X64 = nameof(X64);
+        public static readonly string Arm = nameof(Arm);
+        public static readonly string Arm64 = nameof(Arm64);
```

Comment:
At the moment we don't distinguish between arm and arm64 - I am not sure if we will in the future...